draft-ietf-jsonpath-base.md

```diff
@@ -839,33 +717,29 @@ Queries:
 | :---: | ------ | :----------: | ------- |
 | `$.o['j j']['k.k']`   | `3` | `$['o']['j j']['k.k']`      | Named value in nested object      |
 | `$.o["j j"]["k.k"]`   | `3` | `$['o']['j j']['k.k']`      | Named value in nested object      |
-| `$.a[1]`   | `"b"` | `$['a'][1]`      | Member of array      |
-| `$.a[-2]`   | `"a"` | `$['a'][0]`      | Member of array, from the end      |
 | `$["'"]["@"]` | `2` | `$['\'']['@']` | Unusual member names
-{: title="Index selector examples"}
+| `$.j`   | `{"k": 3}` | `$['j']`      | Named value of an object      |
+| `$.j.k` | `3`        | `$['j']['k']` | Named value in nested object  |
+{: title="Name selector examples"}
 
-### Index Wildcard Selector
+### Wildcard Selector {#wildcard}
 
 #### Syntax
 {: unnumbered}
 
-The index wildcard selector has the form `[*]`.
+The wildcard selector consists of an asterisk.
 
 ~~~~ abnf
-index-wild-selector    = "[" wildcard "]"  ;  asterisk enclosed by brackets
+wildcard = "*"
 ~~~~
 
 #### Semantics
 {: unnumbered}
 
-An `index-wild-selector`
-selects the nodes of all member values of an object as well as of all elements of an
-array.
-
-Applying the `index-wild-selector` to a primitive JSON value (that is,
-a number, a string, `true`, `false`, or `null`) selects no node.
+A `wildcard` selector selects the nodes of all children of an object or array.
```

Comment:
Did we define "children of an object"?
(These are the values, not the members.)

Reply:
Yes we did. The terminology section says this:

> Children (of a node):
>
>    If the node is an array, each of its elements, or if the node is an object, each of its member values (but not its member names). If the node is neither an array nor an object, it has no children.

Reply:
Thanks for looking that up for me 😊